src/tests/specs/smoke/smokeTest.spec.ts

```diff
@@ -79,19 +79,27 @@ describe('Planner Smoke Tests:', () => {
 
   it('Associate/Re-associate workitem with an Iteration', async () => {
     await planner.workItemList.clickWorkItem(c.workItemTitle3);
-    await planner.quickPreview.addIteration(c.iteration1);
+    await planner.quickPreview.addIteration(c.dropdownIteration1);
     expect(await planner.quickPreview.hasIteration(c.iteration1)).toBeTruthy();
     await planner.quickPreview.close();
 
     await planner.workItemList.clickWorkItem(c.workItemTitle3);
     expect(await planner.quickPreview.hasIteration(c.iteration1)).toBeTruthy();
-    await planner.quickPreview.addIteration(c.iteration2);
+    await planner.quickPreview.addIteration(c.dropdownIteration2);
     expect(await planner.quickPreview.hasIteration(c.iteration2)).toBeTruthy();
     await planner.quickPreview.close();
 
     await planner.workItemList.clickWorkItem(c.workItemTitle3);
     expect(await planner.quickPreview.hasIteration(c.iteration2)).toBeTruthy();
     await planner.quickPreview.close();
+
+    await planner.workItemList.clickWorkItem(c.workItemTitle3);
+    await planner.quickPreview.typeaHeadSearch('z');
```

Comment:
@vikram-raj could you add this 'z' in constant.ts

Reply:
@Raunak1203 Done.